test/unit-tests/ad_manager_utils_test.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Unit test class for the Vast Ad Manager
+ * https://github.com/Automattic/expect.js
+ */
+
+var adManagerUtils = require("../../utils/ad_manager_utils.js");
+
+describe('Ad Manager Utility Class:', function()
+{
+
+  it('Time Stamps Should Be Converted to Milliseconds', function()
```

Comment:
Can we add unit tests with improper hh:mm:ss formats? Such as 10:10: or :10:10 or 10:10:10:10?